internal/cli/cmds.go

```diff
@@ -362,10 +362,10 @@ func runRemove(language string, args []string, upgrade bool,
 
 	// Map from normalized package names to original package
 	// names.
-	normPkgs := map[api.PkgName]api.PkgName{}
```

Comment:
any reason to not keep this as api.PkgName?

Reply:
I'm starting a slog towards `api.PkgName` being exclusively used for normalized package names. It bit me a few times in this PR, expecting that values would match when one or the other wasn't normalized. Open for other strategies here, but `string` vs `api.PkgName` seemed like the most sensible option without lots of churn.

Reply:
ah, makes sense :) i miss rust...